src/index.html

```diff
@@ -25,7 +25,7 @@
     <script type="module" src="/assets/kompendium/kompendium/kompendium.esm.js"></script>
     <script nomodule src="/assets/kompendium/kompendium/kompendium.js"></script>
 
-    <link href="/style/color-palette-extended-light-mode-only.css" rel="stylesheet" />
+    <link href="/style/color-palette-extended.css" rel="stylesheet" />
```

Comment:
Do we need to declare a BREAKING CHANGE in this commit message?

Reply:
Not for this specific change at least, since this only applies to the docs. We might need to declare a breaking change because of other changes though. Possibly because of the changes in `color-palette-extended.css`, but that shouldn't really be used by anyone yet, since it wasn't working before now. If anyone was, I guess this is to be considered a major bug fix 😆

Reply:
I'll test this PR with `main` of lime-webclient. That should tell us if there are breaking changes…

Reply:
Thnx. I still have to do many things in the web client PR. But check it out the current. Link in the description.

Reply:
I will test it with the latest `main` of lime-webclient before this PR is merged, as that's what's going to tell us if there are breaking changes. Once this PR is merged, I'll go ahead and test what the dark mode actually looks like in your branch 👍